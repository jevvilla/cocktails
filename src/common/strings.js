export default {
  HOW_TO_PREPARE: 'How to prepate',
  SEARCH_COCKTAIL_PLACEHOLDER: 'Search a cocktail...',
  OVERVIEW_HEADER_TITLE: 'Random drinks 0.1',
  NO_SEARCH_RESULTS: 'No results for: ',
};
