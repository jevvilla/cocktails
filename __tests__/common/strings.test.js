import strings from '../../src/common/strings';

describe('strings', () => {
  test('should match snapshot', () => {
    expect(strings).toMatchSnapshot();
  });
});
