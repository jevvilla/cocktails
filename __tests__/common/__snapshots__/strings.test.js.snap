// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`strings should match snapshot 1`] = `
Object {
  "HOW_TO_PREPARE": "How to prepate",
  "NO_SEARCH_RESULTS": "No results for: ",
  "OVERVIEW_HEADER_TITLE": "Random drinks 0.1",
  "SEARCH_COCKTAIL_PLACEHOLDER": "Search a cocktail...",
}
`;
